sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/EventGridPublisherAsyncClient.java

```diff
@@ -33,6 +36,9 @@
 
     private final EventGridServiceVersion serviceVersion;
 
+    // Please see <a href=https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers>here</a>
+    // for more information on Azure resource provider namespaces.
+
```

Comment:
Add this to the Constants file where the value is declared.

Reply:
I copied the value there but didn't copy the comment ;-(. Corrected.